src/commands/variables.rs

```diff
@@ -74,6 +99,18 @@ pub async fn command(args: Args) -> Result<()> {
         _ => bail!(RailwayError::NoServiceLinked),
     };
 
+    if args.set_from_stdin {
+        let stdin = stdin();
+
+        for line in stdin.lock().lines() {
+            let line = line?;
+
+            if !line.trim().is_empty() {
+                args.set.push(line);
```

Comment:
Not super important b/c we trim later, but maybe this for consistency
```suggestion
                args.set.push(line.trim().to_string());
```